axelor-business-project/src/main/resources/domains/InvoicingProject.xml

```diff
@@ -14,26 +14,28 @@
     <many-to-many name="expenseLineSet" ref="com.axelor.apps.hr.db.ExpenseLine" title="Expense Lines" />
     <many-to-many name="projectSet" ref="com.axelor.apps.project.db.Project" title="Project" />
     <many-to-many name="teamTaskSet" ref="com.axelor.team.db.TeamTask" title="Tasks" />
-		<integer name="logTimesSetPrioritySelect" selection="invoicing.project.priority.select" title="Log Times Priority" default="3"/>
-		<integer name="saleOrderLineSetPrioritySelect" selection="invoicing.project.priority.select" title="Sale order lines Priority" default="1"/>
-		<integer name="purchaseOrderLineSetPrioritySelect" selection="invoicing.project.priority.select" title="Purchase order lines Priority" default="2"/>
-		<integer name="expenseLineSetPrioritySelect" selection="invoicing.project.priority.select" title="Expense Lines Priority" default="4"/>
-		<integer name="projectSetPrioritySelect" selection="invoicing.project.priority.select" title="Project Priority" default="5"/>
-		<integer name="teamTaskSetPrioritySelect" selection="invoicing.project.priority.select" title="Task Priority" default="6"/>
     <many-to-one name="invoice" ref="com.axelor.apps.account.db.Invoice" title="Invoice generated" />
     <date name="deadlineDate" title="Deadline"/>
     <string name="comments" title="Comments" large="true" />
     <boolean name="attachAnnexToInvoice" title="Attach the Annex to the invoice"/>
     <integer name="statusSelect" title="Status" selection="business.project.invoicing.project.status.select" default="1"/>
-    
+    <one-to-many name="invoicingElementOrderLst" title="Invoicing element order" ref="com.axelor.apps.businessproject.db.InvoicingElementOrder" orderBy="sequence"/>
```

Comment:
correct typo its 'invoicingElementOrderList' (List) not 'invoicingElementOrderLst', check all such mistakes